src/encoded/tests/test_types_init_collections.py

```diff
@@ -223,9 +223,9 @@ def vendor_data_alias(lab, award):
         'title': 'Wrong Alias Biochemical',
         'lab': lab['@id'],
         'award': award['@id'],
-        'aliases': ['my_lab:this_is_correct_one',
-                    'my_lab:this/is_wrong',
-                    'my_lab:this\is_wrong_too']}
```

Comment:
Note that this use of backslash was not in a raw string so needed a doubling of the character or an `r'...'`.